Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365_test.py

```diff
@@ -823,4 +837,16 @@ def test_create_message_with_html_body(self, mock_message, mock_body, mock_html_
             name="file.txt", content="data", is_inline=True, content_id="12345"
         )
         mock_message.assert_called_once()
-        self.assertIsInstance(result, MagicMock)
+        assert isinstance(result, MagicMock)
+
+
+@pytest.mark.parametrize("headers, expected_formatted_headers", [
+    ([("Message-ID", '<valid_header>')], [("Message-ID", '<valid_header>')]),
+    ([("Message-ID", '<[valid_header]>')], [("Message-ID", '<valid_header>')]),
```

Comment:
since the `expected_formatted_headers` are the same in both cases (and there are only two), we can remove them from the parametrized part and hardcode them into the test, making the test easier to read. Wdyt?

Reply:
Added another use case with a different `expected_formatted_headers` value so I think it's ok to leave it that way, lmk if you feel differently about this.